css-positioning/absolute.css

```diff
@@ -0,0 +1,36 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  bottom: 70%;
+  left: 74%;
```

Comment:
Please revise this to use the `top` and `right` properties, as discussed in the Q&A